frontend/src/pages/Validators.js

```diff
@@ -113,12 +126,15 @@ export default function Validators() {
         break;
       case "redelegate":
         let isValidRedelegation = false;
-        if (delegations?.delegations.length > 0) {
-          for (let i = 0; i < delegations?.delegations.length; i++) {
-            let item = delegations?.delegations[i];
+        if (delegations?.delegations.delegations.length > 0) {
```

Comment:
you can assign to delegations to one variable and you can take it from variable

Reply:
done